NoteFind/View Controllers/NewNoteViewController.m

```diff
@@ -93,14 +92,20 @@ - (void)addBooks:(nonnull NSMutableArray *)selectedBooks {
     NSLog(@"books added: %@", self.tags);
 }
 
-- (IBAction)importNote:(id)sender {
 
+
+- (void)showGallery {
```

Comment:
Is this and the next method needed here or are they contained within the camera importer?